content/maintainers-guide/reviewing-pull-requests.md

```diff
@@ -1,156 +1,215 @@
+- Add the label <label type="keyfeature">Key feature</label> if the Pull Request must be mentioned in the Release Note.
+- Add the label <label type="bcbreak">BC break</label> if the Pull Request introduces a [BC Break][bc-break].
+- Add the label <label type="needsdoc">Needs documentation</label> if the Pull Request introduces changes that need to be documented in [DevDocs][dev-doc].
+
+These actions are very important as they will be key to writing a good Release Note and ChangeLog for the next version.
+
+#### Documentation
+
+Some Pull Request contain changes that need an update of the developer documentation.
 
-The items above are very important as they will be key to writing a good Release Note and ChangeLog for the next version.
+Example of such Pull Requests:
+- If it contains [BC breaks][bc-break] or deprecations, they must be listed, including recommended workarounds (if any).
+- If it introduces or modifies a component or subsystem — especially when theme or module developers are expected to use it.
 
-It is also recommended to :
+#### Final actions
 
+- If the Pull Request is related to an issue, check whether the issue is fixed and closed and whether it should be: some issues can get closed automatically even though the Pull Request only addresses part of the issue.
 - Thank the Pull Request author and anybody else who invested notable energy into the Pull Request (code review, code suggestions, QA validation, usecase specification ...).
-- If the Pull Request is related to an issue, check whether the issue is fixed and closed (it might be done automatically depending on the Pull Request) and whether it should be.
 
-### Pull Requests that need an update of the [developer documentation][dev-doc]
 
-If a Pull Request contains changes that need an update of the developer documentation, please add the label "Needs documentation" when the PR is accepted.
+![Thank you message](../images/thank-you.png)
 
-Later, when the documentation has been updated, we can remove the previous label and add "Documentation ✓".
+## Labeling reference
 
-Example of such Pull Requests:
-- If it contains [BC breaks][bc-break], BC breaks need to be listed
-- If it introduces a new component or system, it must be documented, especially if we expect theme or module developers to use it
-- If it modifies an item document, so that the documention becomes erroneous or outdated
+Labeling helps project members keep track of the status of Pull Requests.
+
+Two bots monitor the project's Issues and Pull Requests on GitHub to help automate common tasks.
+
+- [Prestonbot][prestonbot] will evaluate whether there are missing/invalid items in Pull Requests' descriptions and label them accordingly.
+- [Issuebot][issuebot] helps sync the status of Issues with its linked Pull Request by applying labels and performing kaban board automation.
+
+### Action required
+
+The following labels are added when an action is required:
```

Comment:
What about the label `Waiting for dev`?

Reply:
In which case you would need it in a PR?

Reply:
We use it when a developer action is needed, excluding review. For example if CI is failing for an unknown reason or specific expertise is required. I am not sure it's relevant to write it in this guide.